kiwipy/rmq/communicator.py

```diff
@@ -503,6 +504,15 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     def loop(self):
         return self._loop
 
+    def is_closed(self) -> bool:
+        return self._closed
+
+    def close(self):
+        if self.is_closed():
+            return
+        self.stop()
+        self._closed = True
```

Comment:
Do we also want to clean up the `RmqCommunicator` instance here?

Reply:
Accepted.  Clean up the loop here and `_stop_signal` in `start()`